projects/nDPI/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM sweetvishnya/ubuntu20.04-sydr-fuzz
+
+RUN apt-get update && apt install build-essential git gettext \
+                      flex bison libtool autoconf automake pkg-config \
+                      libpcap-dev libjson-c-dev libnuma-dev libpcre2-dev \
```

Comment:
You should also build libpcap and json-c with instrumentation as they do in [OSS-Fuzz](https://github.com/google/oss-fuzz/blob/master/projects/ndpi/build.sh).